styleguide/source/_patterns/01-molecules/10-search-results/00-search-result/search-result-external.twig

```diff
@@ -0,0 +1,13 @@
+<div class="search-result-external {{ class }}">
+   <a href="{{ url }}" class="search-result_primary-wrapper">
+     <div class="search-result_label">
+         {% include 'atoms-title-label' with { 'content' : 'AMA Wire', 'class': 'search-result' } %}
+         {% include 'atoms-arrow-external' %}
+     </div>
+     {% include 'atoms-h4' with { 'class' : 'search-result_title', 'content' : 'Visit AMA Wire to view news and more information related to this topic.' } %}
+   </a>
+     <div class="search-result_members-only-label-wrapper">
```

Comment:
In the comps on hover all text in the row turns blue. Possible to include this in the `<a>` tag in order to have a cohesive hover interaction?